pkg/frontend/admin_openshiftcluster_kubernetespods_logs.go

```diff
@@ -0,0 +1,55 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+)
+
+func (f *frontend) getAdminKubernetesPodLogs(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	log := ctx.Value(middleware.ContextKeyLog).(*logrus.Entry)
+	r.URL.Path = filepath.Dir(r.URL.Path)
+	b, err := f._getAdminKubernetesPodLogs(ctx, r, log)
+
+	adminReply(log, w, nil, b, err)
```

Comment:
What content type do we return in responce?

Reply:
Plain text and errors in json